src/cmd/linuxkit/moby/build.go

```diff
@@ -116,7 +116,8 @@ func outputImage(image *Image, section string, prefix string, m Moby, idMap map[
 }
 
 // Build performs the actual build process
-func Build(m Moby, w io.Writer, pull bool, tp string, decompressKernel bool, cacheDir string, dockerCache bool, arch string) error {
+func Build(m Moby, w io.Writer, opts BuildOpts) error {
+	//pull bool, tp string, decompressKernel bool, cacheDir string, dockerCache bool, arch string
```

Comment:
I think this can go

Reply:
Oh, yeah, that was to make it easier for me to convert things. Oops.